qbench/cli.py

```diff
@@ -2,9 +2,11 @@
 from argparse import ArgumentParser
 
 from .fourier import add_fourier_parser
-from .logging import configure_logging
+from .fourier_certification import add_fourier_parser_certification
```

Comment:
Maybe this should be named `add_fourier_certification_parser`. The current name suggests that this is a parser certification, which it is not. It is a parser for Fourier certification task.

Reply:
Commit 273f78b